scripts/generate-openapi-script.sh

```diff
@@ -19,8 +17,8 @@ rm -Rf src/app/shared/openapi
 
 if [ "$npm_package_config_use_circle" = true ]
 then
-        SWAGGER_PATH="${CIRCLE_CI_PATH}""/swagger.yaml"
```

Comment:
codacy seems to be warning about removing the quotes here, didn't read too deeply into it, might be worth a look